A6lib.cpp

```diff
@@ -441,3 +441,31 @@ byte A6lib::A6waitFor(const char *resp1, const char *resp2, int timeout, String
     }
     return retVal;
 }
+
+
+// making modem functioning or free from previous engagements
+int A6lib::getFree(long timeout)
+{
+	byte retval=A6_NOTOK;
+    logln(F("Setting module free from past command..."));
+	//long ptime=millis();
```

Comment:
yeah, I see that as a very common problem with my code.  I will try to keep this in mind for future.